src/main/java/appeng/core/Registration.java

```diff
@@ -225,26 +201,13 @@ private void registerSpatialDimension()
 	private void registerCraftHandlers( final IRecipeHandlerRegistry registry )
 	{
 		registry.addNewSubItemResolver( new AEItemResolver() );
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this use of "addNewSubItemResolver"; it is deprecated. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)